pyvista/plotting/mapper.py

```diff
@@ -298,10 +312,20 @@ def configure_scalars_mode(
             else:
                 self.SetColorModeToMapScalars()
 
+            return new_scalars_name, assoc
+
         def set_custom_opacity(
             self, opacity, color, mesh, n_colors, preference, interpolate_before_map, rgb, theme
         ):
-            """Set custom opacity."""
+            """Set custom opacity.
+
+            Returns
+            -------
+            str or None
+                If the scalars do not exist within the dataset, this is the
+                name of the scalars array.
+
```

Comment:
This returns a tuple now, looking at `configure_scalars_mode()`.

Reply:
Fixed.